### ISSUE TITLE
Unfork gsscoder/commandline

### ISSUE THREAD

**Issue by ericnewton76** [commandlineparser/commandline#8]
@nemec and et al

Should we consider de-forking from gsscoder/commandline ?   Ironically git still provides a way to reintegrate everything later.

It involves deleting this repo and recreating from the commits.  (Losing issues again, ugh)

**Comment by nemec** [comment]
I guess it would remove all the "forked from" messages. Does a fork cause other challenges vs. a new repo?

It looks like we could import directly from the old repository into a new one: https://help.github.com/articles/importing-a-repository-with-github-importer/

**Comment by ericnewton76** [comment]
Yeah.  Did that, created commandlineparserX and cloning issues into there, will shut this one down soon

**Comment by ericnewton76** [comment]
Interesting.  We lose the whole fork tree.
